pkg/products/observability/reconciler.go

```diff
@@ -224,6 +441,9 @@ func (r *Reconciler) reconcileComponents(ctx context.Context, serverClient k8scl
 		return integreatlyv1alpha1.PhaseInProgress, err
 	}
 	if op == controllerutil.OperationResultUpdated || op == controllerutil.OperationResultCreated {
+		//return integreatlyv1alpha1.PhaseInProgress, nil
+		r.log.Info(string(integreatlyv1alpha1.PhaseCompleted))
+	} else {
```

Comment:
this else will cause the reconciler to always return a PhaseInProgress if there is no update to the cr. 
The initial idea here was to return an inProgress if we make some change to the observability cr to allow the oo time to make the updates. But it's probably not the best approach. There is a phase reported the oo which is probably a more controlled way to report back the oo progress to the rhmi cr.